### ISSUE TITLE
Attaching crashes minimal WKWebView app (jailed iOS 26, JavaScriptCore libpas scavenger, SIGILL)

### ISSUE THREAD

**Issue by vnc0** [frida/frida-core#1250]
Attaching Frida to an app with an active `WKWebView` on a physical iOS device running **iOS 26.2** causes the target process to crash approximately 1–3 seconds later. 

This happens even during a bare `attach()` and immediate `detach()` without injecting any scripts. The target process terminates with `EXC_BAD_ACCESS` / `SIGILL` on the `JavaScriptCore libpas scavenger` thread. Apps without a `WKWebView` attach normally and remain completely stable.

## Environment

* **Frida:** 17.16.2 & 17.16.4 (latest)
* **Device:** iPad (A16, `iPad15,7`), **iOS 26.2 (23C55)**
* **Setup:** Jailed device; Frida injected via `debugserver` into a dev-signed (`get-task-allow`) app over USB
* **Host:** macOS 26.5

## Steps to Reproduce

[WebCrashRepro.zip](https://github.com/user-attachments/files/30314976/WebCrashRepro.zip)

1. Launch a minimal app just containing a single full-screen `WKWebView`.
2. Run a simple attach and detach sequence:

```python
import frida

dev = frida.get_usb_device()
pid = dev.get_process("WebCrashRepro").pid
dev.attach(pid).detach()  # Target crashes ~1–3s later without loading any scripts
```

*Note: Running `frida -U -n WebCrashRepro` via the CLI and exiting immediately produces the same crash.*

## Observed Behavior

The target process crashes shortly after detaching:

```text
exception: EXC_BAD_ACCESS  signal: SIGILL  (Illegal instruction: 4)
faulting thread 2: JavaScriptCore libpas scavenger
  ?                            +4713076124  
  libsystem_platform.dylib     +16728  _sigtramp
  ?                            +4713093764  
  libsystem_platform.dylib     +16728  _sigtramp
  ?                            +4713093764  
  libsystem_platform.dylib     +16728  _sigtramp
  ?                            +4713093764  
  libsystem_platform.dylib     +16728  _sigtramp
  ?                            +4713093764  
  libsystem_platform.dylib     +16728  _sigtramp
  ?                            +4713093200  
  ?                            +4713093720  
  libsystem_platform.dylib     +16728  _sigtramp
  JavaScriptCore               +58608  stop_allocator
  JavaScriptCore               +54880  pas_thread_local_cache_for_all
  JavaScriptCore               +53068  scavenger_thread_main
  libsystem_pthread.dylib      +17484  _pthread_start
  libsystem_pthread.dylib      +2252  thread_start
```

The app’s main thread is idle in its run loop at the time of the fault. Because the Frida session has already detached, no `frida` or `gum` images appear in the crash log. The attach process seems to leave WebKit's JavaScriptCore `libpas` heap corrupted, triggering the crash as soon as the scavenger thread next executes.

## Expected Behavior

Attaching to and detaching from an app hosting a `WKWebView` should not corrupt the JavaScriptCore heap or destabilize the target process.

## Additional Context & What Has Been Ruled Out

The crash still occurs under the following conditions:
* **Without script execution:** A bare `attach()` + `detach()` triggers the issue without walking objects or loading scripts.
* **Persistent sessions:** Holding a single attach session open fails the same way.
* **JIT disabled:** Launching the app with `JSC_useJIT=0` makes no difference.
* **Frida version differences:** Reproduces on both `17.16.2` and `17.16.4`.

### Comparison
Testing the exact same app build and Frida setup on an iPhone running **iOS 15.8.7** (`iPhone9,4`) is completely stable across multiple attach, script-load, and read cycles. This suggests the issue is isolated to WebKit / JavaScriptCore `libpas` behavior on iOS 26.
